test/run-tests

```diff
@@ -952,6 +986,12 @@ def main():
         default=int(os.environ.get("TEST_HARNESS_MAX_NUM_STATUSES", "60")),
         help="Number of statuses to retrieve - default 60 - github default is 30",
     )
+    parser.add_argument(
+        "--openstack",
```

Comment:
How about making this an argument like "--backend" that defaults to something like "kvm" and also accepts "citool". Then you do checks later like:

```
if args.backend == KVM_BACKEND:
    ...
elif args.backend == CITOOL_BACKEND:
    ...
```

This avoids checks like 


```
if not openstack:
    ....
```

that are less clear to understand.

Reply:
Thanks, added